### ISSUE TITLE
Update region names

### ISSUE THREAD

**Issue by MartinKolarik** [jsdelivr/globalping#179]
Missed part of #162, "southern america" should be "South America"

**Comment by patrykcieszkowski** [comment]
Why particularily `South`, not `Southern`?

**Comment by jimaek** [comment]
That's the most popular name I think. But we can support both

**Comment by patrykcieszkowski** [comment]
we match the region name based on country-code, so technically speaking we can name them whatever we want.

**Comment by MartinKolarik** [comment]
> Why particularily `South`, not `Southern`?

I think "South" is the correct version of the two but didn't actually change that on purpose, it must have been Grammarly fixing that for me.

**Comment by ayuhito** [comment]
I threw it in a couple gazetteers and there's a clear distinction between South America and Southern America (which is in North America). 

They don't mix really.

**Comment by patrykcieszkowski** [comment]
I mean, South America is the continent, Southern America is more general, describing the region.

**Comment by MartinKolarik** [comment]
> I mean, South America is the continent, Southern America is more general, describing the region.

Yes, but which region 😄 as @ayuhito points out, "Southern" is also used to refer to the South part of the US, which is an entirely different region.

**Comment by patrykcieszkowski** [comment]
hm, ok

**Comment by MartinKolarik** [comment]
But I'm not sure how the original list of regions was constructed. Specifically for South America, it seems there's no point in having it, since that's already a continent?

**Comment by patrykcieszkowski** [comment]
`northern-europe`, `western-europe`, `eastern-europe` etc aren't seperate continents, and yet it might be useful to have a filter targeting them.

https://github.com/jsdelivr/globalping/blob/master/src/lib/location/regions.ts

**Comment by ayuhito** [comment]
It feels a little arbitrary. What about matching Cloudflare if we want to have that much specificity? https://developers.cloudflare.com/load-balancing/reference/region-mapping-api/

Otherwise, we can have a broader set of regions matching the data center tab: https://www.cloudflare.com/network/

**Comment by MartinKolarik** [comment]
Sure, I'm not questioning regions as a whole, just Southern America specifically. E.g. https://en.wikipedia.org/wiki/Northern_America is a used term with a clear definition, but the Southern version isn't, and for South, there's already a continent.

**Comment by patrykcieszkowski** [comment]
> It feels a little arbitrary. What about matching Cloudflare if we want to have that much specificity? https://developers.cloudflare.com/load-balancing/reference/region-mapping-api/
> 
> Otherwise, we can have a broader set of regions matching the data center tab: https://www.cloudflare.com/network/

Well, it is arbitrary. But I see no reason to rely on 3rd party API for such a simple mechanism.

**Comment by patrykcieszkowski** [comment]
> Sure, I'm not questioning regions as a whole, just Southern America specifically. E.g. https://en.wikipedia.org/wiki/Northern_America is a used term with a clear definition, but the Southern version isn't, and for South, there's already a continent.

I'm out when it comes to geographical divagations. We can call it Latin America, South America, I really dont mind.

**Comment by ayuhito** [comment]
> Well, it is arbitrary. But I see no reason to rely on 3rd party API for such a simple mechanism.

I didn't mean use their API. I meant match their regions and region codes since it seems like a solid option of choices that cover most usecases with correct names.

**Comment by MartinKolarik** [comment]
For now, let's do the pretty names and normalization, then we can discuss the actual list of options.

**Comment by patrykcieszkowski** [comment]
in terms of filter matching, do we want dash or space seperation for exact matches?

**Comment by MartinKolarik** [comment]
As a user, I should write spaces, same as for cities (except magic mode).
